Server/DeviceManagement/DeviceManagement.Application/QueryResponses/GetUserQueryResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using ModularHouse.Libraries.InternalMessaging.CQRS.Abstractions.Query;
+
+namespace ModularHouse.Server.DeviceManagement.Application.QueryResponses;
+
+public record GetUserQueryResponse(Guid Id, DateTime AdditionDate) : IQueryResponse;
```

Comment:
It seems a bit strange, that `QueryResponse` contains data of `User`;
I think it must be something like `queryResponse.User.Id`, but not `queryResponse.Id`
Maybe it make sense to create additional `UserDto`? And `GetUserQueryResult` would contain `UserDto User` as property
Let me know what you think